extract/bobaedream/main.py

```
@@ -30,30 +37,70 @@ def _save_to_s3(post: dict, bucket: str, key: str):
-            car_id=car_id, date=date, batch=batch, community=COMMUNITY, post_id=post_info["id"]
+            car_id=car_id,
+            date=date,
+            batch=batch,
+            community=COMMUNITY,
+            post_id=post_info["id"],
         )
+
+        # S3에 저장 시도
         try:
             _save_to_s3(post, bucket, s3_key)
         except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)